travis/scripts/05-run.sh

```diff
@@ -94,6 +94,7 @@ if [ "$RUN_APP" == 1 ]; then
         cd "$UAA_APP_FOLDER"
         java -jar target/*.war \
             --spring.profiles.active="$PROFILE" \
+            --logging.level.io.github.jhipster=ERROR \
```

Comment:
in the latest Spring Boot 2 libs there seems to be a dependency when creating the log config tree. The keys must be in the correct hierarchical order or it results in mapping exceptions